src/chanakya/web/routes.py

```
@@ -340,17 +309,11 @@ async def record():
             )
         except RuntimeError as e:
             if "Event loop is closed" in str(e):
-                app.logger.error(
-                    f"EVENT LOOP CLOSED ERROR in /record: {e}", exc_info=True
-                )
-                return jsonify(
-                    {"response": "Internal server error: Event loop issue."}
-                ), 500
+                app.logger.error(f"EVENT LOOP CLOSED ERROR in /record: {e}", exc_info=True)
+                return jsonify({"response": "Internal server error: Event loop issue."}), 500
             else:
```

Ruff (0.15.6)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)